packages/wrangler/src/pages/build.ts

```diff
@@ -121,32 +134,110 @@ export const Handler = async ({
 		}
 	}
 
+	// for now let's restrict `outfile` to `_worker.bundle` if the
+	// experimentalWorkerBundle flag is set. Once we remove the flag
+	// this will no longer be an issue
+	outfile = experimentalWorkerBundle ? "_worker.bundle" : outfile;
```

Comment:
```suggestion
```

I wouldn't bother with this.

Reply:
the default `outfile` is `_worker.js` atm. If I change that to `_worker.bundle` it'll break crr behaviour. I remember saying don't want to break crr behaviour?

Reply:
I'll manually specify `--outfile=_worker.bundle`. I put it in a known location anyway.

Reply:
✅